CONTRIBUTING.md

```diff
@@ -37,7 +37,7 @@ Preamble: If you're interested in to contribute to the ml5 project, just know yo
 3. you make a comment on an existing issue or post your issue and indicate that you're curious to do your best to solve it 🔬
 4. you create a new branch on your `forked` copy of the ml5-library and call it something meaningful like `fix-detection-results`
 5. you jam on fixing the bug, commit your changes with meaningful commit messages, and push your changes to your bug fix branch (e.g. `fix-detection-results`)
-6. when ready, make a pull request to the `release` branch or `development` branch of ml5-library. Submit to the `release` branch if the bug you found was in the current ml5 release - the version of ml5 which is on npm. Submit to the `development` branch if the bug you found was in `development` branch of ml5-library.
+6. when ready, make a pull request to the `main` branch of ml5-library. Prepend "[NEEDS RELEASE]" to the title if the bug you found was in the current ml5 release - the version of ml5 which is on npm.
```

Comment:
We can also create a label which gets assigned if the maintainers agree with or want to add the `needs release` tag.

Reply:
Agreed!

Reply:
Went with something like `ready for release` for PRs that need to be released and have received approval. Can come back and change this process as we start to see it in action!